add spring database connection

Akim0982 opened the pull request.

Commit:
add spring database connection

DmitrySay commented on src/main/resources/application.properties:
```diff
@@ -1 +1,4 @@
-
+spring.datasource.driver-class-name=
```
add Postgres driver value

Commit:
add postgresDriver

Commit:
add in AuthorController: GetMapping, PostMapping, PutMapping, DeleteMapping
add postgresSQL

Commit:
add in AuthorController: GetMapping, PostMapping, PutMapping, DeleteMapping
add postgresSQL

Akim0982 committed:
Delete BookController.java

delete book

Akim0982 committed:
Delete Book.java

delete book

Commit:
1. add in AuthorController: annotation Transactional, delete from @GetMapping("") - (""), change "theAuthor" to "author"
2. change method "deleteAuthor" in model(Author).
3. add exception in method "findById" in service(AuthorService), but he doesn't work...
4. application.properties: delete - spring.jpa.database=postgresql, change database, delete - spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL82Dialect
5. add file - init(postgresql).sql and init(MySQL).sql

Commit:
Merge remote-tracking branch 'origin/feature/application' into feature/application

Akim0982 commented:
1. add in AuthorController: annotation Transactional, delete from @GetMapping("") - (""), change "theAuthor" to "author"
2. change method "deleteAuthor" in model(Author).
3. add exception in method "findById" in service(AuthorService), but he doesn't work...
4. application.properties: delete - spring.jpa.database=postgresql, change database, delete - spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL82Dialect
5. add file - init(postgresql).sql and init(MySQL).sql
